storage/src/testFixtures/java/tech/pegasys/teku/storage/api/StubEth1DepositChannel.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.storage.api;
+
+import com.google.common.primitives.UnsignedLong;
+import tech.pegasys.teku.datastructures.blocks.Eth1BlockData;
+import tech.pegasys.teku.datastructures.operations.DepositWithIndex;
+
+public class StubEth1DepositChannel implements Eth1DepositChannel {
```

Comment:
We don't really need this - you can just use a standard mock (and it's currently unused anyway)